src/computation/logic/distribute-levelling-seats.ts

```diff
@@ -85,22 +85,21 @@ function nationalDistributionFilter(
 ): NationalDistributionResult {
     let totalVotes = 0;
     let seatsToDistribute = payload.levelingSeats;
-    const levelingParties: Result[] = [];
+    const levelingParties: Dictionary<PartyResult> = {};
     for (const partyCode of levelingPartyCodes) {
         totalVotes += partyResults[partyCode].votes;
         seatsToDistribute += partyResults[partyCode].districtSeats;
-        const party: Result = {
-            countyId: -1,
-            electionId: -1,
-            partyId: -1,
-            resultId: -1,
-            countyName: "",
+        const party: PartyResult = {
```

Comment:
Because they are typed as number or string. Retyping them all as number | undefined, and constantly checking and handling the undefined case is a hassle.